src/axom/inlet/Table.cpp

```diff
@@ -127,26 +155,25 @@ Verifiable& Table::addStringArray(const std::string& name,
 Table& Table::addGenericArray(const std::string& name,
                               const std::string& description)
 {
-  if(m_sidreGroup->hasView(ARRAY_INDICIES_VIEW_NAME))
+  if(isGenericContainer())
   {
-    SLIC_ERROR(
-      fmt::format("[Inlet] Adding array of structs to array of structs {0} is "
-                  "not supported",
-                  m_name));
+    SLIC_ERROR(fmt::format(
+      "[Inlet] Adding array of container to array of structs {0} is "
```

Comment:
There is something wrong with this error message.  Is is supposed to be "Adding array of containers"?

Reply:
Was supposed to be analogous to the message in its dictionary counterpart - "Adding array of structs to container of structs"